lib/rules/best-practises/no-empty-blocks.js

```diff
@@ -16,32 +20,53 @@ const meta = {
           description: 'empty block in constructor without parent initialization',
           code: 'constructor () {}',
         },
+        {
+          description: 'empty modifier with only _ (when allowEmptyModifiers is false)',
+          code: 'modifier onlyOwner { _; }',
+        },
       ],
       good: [
         { description: 'empty receive function', code: 'receive () external {}' },
         { description: 'empty fallback function', code: 'fallback () external {}' },
         {
-          description: 'empty constructor with member initialization list',
-          code: 'constructor(uint param) Foo(param) Bar(param*2) {}',
+          description: 'empty constructor with parent init',
+          code: 'constructor(uint x) Base(x) {}',
         },
+        { description: 'empty modifier with _ allowed', code: 'modifier onlyOwner { _; }' },
+        { description: 'empty try-catch block allowed', code: 'try foo() {} catch {}' },
```

Comment:
given the current rule implementation, this would cause an error on the empty  `try`